documentation/src/main/asciidoc/topics/ref_rest_rolling.adoc

```diff
@@ -12,7 +12,7 @@ POST /v2/caches/{cacheName}/rolling-upgrade/source-connection
 You must provide a `remote-store` definition in JSON format as the body:
 
 .JSON
-[source,json,options="nowrap",subs=attributes+,role="secondary"]
```

Comment:
@gustavonalle and @ryanemerson I probably should have caught this in PR review but, purely as FYI, this role attribute for code blocks needs to be declared correctly or it just grinds down the browser when you open the rendered HTML. Maybe we need a linter for docs to catch this stuff. And I'll report it as an issue in the repo for the maven plugin we use for globbing code blocks.